README.md

```diff
@@ -1,24 +1,36 @@
-While researching this, everything I found led me to believe that Intel Quicksync would not work until we got FreeBSD 12. After researching forums online, I have gotten it to work on 11.3. I am not much more than a noob that is only tinkering but this works for me. I am using a i7 4790K and using Quicksync for 1 1080p to 720p or 1080p to 1080p stream leaves my cpu usage at 1%. I have not tested much more yet.
+##### How to guide for getting Intel Quicksync working on Freenas
+While researching this, everything I found led me to believe that Intel Quicksync would not work until we got FreeBSD 12. After researching forums online, I have gotten it to work on 11.3. I am not much more than a noob that is only tinkering but this works for me.
+## Test Results
+- I am using a i7 4790K and using Quicksync for 1 1080p to 720p or 1080p to 1080p stream leaves my cpu usage at 1%.
+- Core 4th Gen (Haswell) added H.265 and VC1 support in QuickSync. For 1080p transcode, "Haswell or better" is reasonable guidance.
+- Testing on a 1225v6 Xeon (7th Gen Kaby Lake) used ~4% CPU for a 4k HDR -> 1080p SDR hardware transcode. The same machine was at 100% CPU for a software transcode of 4k -> 1080p. Running four 4k hardware transcodes at once did not significantly tax the machine, further testing to determine the max number of possible 4k transcodes was not done.
+- Testing on a G4560 Pentium (7th Gen Kaby Lake) used ~30% CPU for a 4k -> 1080p transcode.
+- Testing on a Core i3 6100T (6th Gen SkyLake) shows that acceleration of a 4k -> 1080p transcode is minimal on it and takes 95% CPU. This is likely due to limited HEVC features in that generation.
+- Testing on a 1225v6 Xeon (7th Gen Kaby Lake) used ~2-3% CPU for a 1080p VC-1 -> 1080p H.264 hardware transcode. The same machine was at 70-75% CPU for a software transcode of VC-1 -> H.264. Some Bluray disks used VC-1, before H.264 became ubiquitous.
+- The Kaby Lake iGPU improved HEVC acceleration and added 10-bit (HDR) HEVC. The HEVC (H.265) codec is used on UHD Blurays. Consider a Kaby Lake or later CPU a requirement for significant acceleration of HEVC (Bluray 4k UHD) content. Note that without HDR10 tone mapping, 4k HDR transcode remains suboptimal, and it is often better to prepare a good SDR 1080p file ahead of time. HDR10 tone mapping is not yet available in a Desktop / Server CPU/iGPU combination.
 ## Compatibility
-- The current (11.3-U1) Freenas Drivers support Intel CPU Generation 2-7
-- The current (11.3-U1) Freenas Drivers DO NOT support Intel CPU Generation 8 and 9
-- Support for Generation 8 and 9 can not be added without a kernel upgrade to 12.0. Don't bother trying to compile it from source.
+- The current (11.3-U1) Freenas Drivers support Intel CPU Generation 2-7, Sandy Bridge through Kaby Lake
+- The current (11.3-U1) Freenas Drivers DO NOT support Intel CPU Generation 8 and 9. This means Coffee Lake and later, Intel Xeon E-xxxxG and Intel Core i3/i5/i7 8xxx and later.
```

Comment:
We can update the U reference here.
Thanks for improving my reference.

Reply:
Done. Calling it Ux now so we don't have to keep updating this. 11.3 won't ever gain 8/9 gen drivers.

Reply:
This, no U* version will... thanks :)